snippet-extractor-output/snippets.yaml

```diff
@@ -2424,10 +2424,10 @@ Word.InlinePicture.getBase64ImageSrc:
 Word.Section.getHeader:
   - |-
     await Word.run(async (context) => {
-        context.document.sections.getFirst()
-            .getHeader("Primary").insertText("This is a header", "Start");
+      context.document.sections.getFirst().getHeader("Primary")
+        .insertParagraph("This is a header", "End");
```

Comment:
any reason for the change from "Start" to "End"?

Reply:
@ElizabethSamuel-MSFT Not a strong one. I wanted to show the header doesn't have to be at the start. I'm happy to change it back.